contracts/KSElasticLMV2.sol

```diff
@@ -7,15 +7,15 @@ import {ReentrancyGuard} from '@openzeppelin/contracts/security/ReentrancyGuard.
 import {EnumerableSet} from '@openzeppelin/contracts/utils/structs/EnumerableSet.sol';
```

Comment:
Let's not use paused and unpaused here, should just a PR to update KyberSwapRole first.
I think only allowing to claim rewards/fees or withdraw liquidity when unpaused is not a good design (shouldn't block users from these actions)